docs/patterns/card/patterns/definition-list.html

```diff
@@ -0,0 +1,60 @@
+<rh-card class="example-dl">
+  <img src="../charmander.png" alt="Charmander Pokémon" slot="image">
```

Comment:
Could you please add one more example where image is not present?
So the card contains just facts (DL) and no (IMAGE) tag?

Reply:
The reason is that we would need to cover both scenarios where the same card may include an image and may not. So having 1 implementation covering both scenarios would be ideal.
Thanks!